ml_service/pipelines/diabetes_regression_build_train_pipeline_with_r.py

```diff
@@ -31,7 +31,7 @@ def main():
     # Make sure to include `r-essentials'
     #   in diabetes_regression/conda_dependencies.yml
     environment = get_environment(
-        aml_workspace, e.aml_env_name, create_new=False)  # NOQA: E501
+        aml_workspace, e.aml_env_name, create_new=e.rebuild_env == "true")  # NOQA: E501
```

Comment:
Please remove the `== "true"` so it can just be whatever the actual value is

Reply:
Hmm I don't understand: the environment variables are not well-typed (i.e. they are strings). In that case the flag would evaluate to true all the time (as a non-empty string). Just like [here](https://github.com/microsoft/MLOpsPython/blob/e6dbfb10545905881e46e6d76855538ccd5add13/diabetes_regression/evaluate/evaluate_model.py#L122)

Reply:
I'd have to check with an actual run or dig into the SDK, but I'm betting the `create_new=False` will be ok with `False` as a string (it's not checking it as a boolean).

Reply:
so the variable should be boolean, (see [here](https://github.com/microsoft/MLOpsPython/blob/e6dbfb10545905881e46e6d76855538ccd5add13/ml_service/util/manage_environment.py#L19)) and Python will convert any type to boolean, but with unexpected results :) 

so the check for the "true" string is vital here for this to work

Reply:
An alternative solution could be to use a library like https://pypi.org/project/environs/ to type the environment variables properly.

Reply:
Environs looks cool, but I don't really want to introduce a new dependency right now.  @sudivate thoughts?

Reply:
Looks good to me, for the scope of this PR lets typecast the string to bool in inside env_variables.py